docs/modules/converter/pages/syntax/auto-animate.adoc

```diff
@@ -0,0 +1,152 @@
+= Auto Animate
+
+This feature is a wrapper of reveal.js auto animation.
+
+Since your adoc file will be transformed into HTML slide file, 
+all settings will be mapped to `data-*` HTML attributes.
```

Comment:
That's not completely true. I think we should talk in general terms like:

> You can define Asciidoctor attributes, options and IDs to configure the reveal.js auto-animate feature.

Reply:
Text revised.